backend/homework/9/src/main/java/com/example/demo/repository/Library.java

```diff
@@ -0,0 +1,93 @@
+package com.example.demo.repository;
+import com.example.demo.exceptions.custom.BadRequestException;
+import com.example.demo.exceptions.custom.ResourceNotFoundException;
+import com.example.demo.entity.Book;
+import org.springframework.stereotype.Repository;
+import java.util.ArrayList;
+import com.example.demo.logger.LogBack;
+import java.util.List;
+@Repository
+public class Library {
```

Comment:
Clean the code
1. Better indentation
2. Modularise in more null-checks/validation functions, utility functions etc
3. Javadocs